### PR TITLE
Modify classes to use a common datetime format

### PR DESCRIPTION
**Pull request by limdylan**

### CODE REVIEW

**Comment by limdylan** [comment]
the formats for date and datetime can be found in `seedu.address.commons.util.DateTimeUtil`

File: src/main/java/seedu/address/commons/util/DateTimeUtil.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * Contains standard datetime and date patterns to be used in Pet Store Helper.
+ */
+public class DateTimeUtil {
+
+    // datetime-related
+    public static final String DATETIME_PATTERN = "d/M/yyyy HHmm";
+    public static final DateTimeFormatter DATETIME_FORMAT =
+            DateTimeFormatter.ofPattern(DateTimeUtil.DATETIME_PATTERN);
```
Comment (tianqi77) [inline src/main/java/seedu/address/commons/util/DateTimeUtil.java]: Maybe can delete the "DateTimeUtil." inside the bracket.

File: src/main/java/seedu/address/commons/util/DateTimeUtil.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    // date-related
+    public static final String DATE_PATTERN = "d/M/yyyy";
+    public static final DateTimeFormatter DATE_FORMAT =
+            DateTimeFormatter.ofPattern(DateTimeUtil.DATE_PATTERN);
```
Comment (tianqi77) [inline src/main/java/seedu/address/commons/util/DateTimeUtil.java]: and maybe for this part also.

**Comment by Zhu-Ruicong** [comment]
LGTM

**Pull request merged**